README.md

```diff
@@ -92,7 +104,10 @@ Installing certificates, and configuring the docker overlay:
 4. Make certs:
     1. `cd ~/.dinghy/certs`
     2. `mkcert -cert-file app.lara.docker.crt -key-file app.lara.docker.key app.lara.docker`
-    3. `mkcert -cert-file app.portal.docker.crt -key-file app.portal.docker.key app.portal.docker`
+    3. `mkcert -cert-file learn.dev.docker.crt -key-file learn.dev.docker.key learn.dev.docker`
```

Comment:
```suggestion
    3. `mkcert -cert-file app.portal.docker.crt -key-file app.portal.docker.key app.portal.docker`
```
A developer will still want to use `app.portal.docker` for the domain.

Reply:
Fixed it.